.github/actions/gcloud/action.yml

```diff
@@ -0,0 +1,21 @@
+name: gcloud
+description: Configure gcloud
+inputs:
+  cloud_sdk_service_account_key:
+    required: true
+  gcp_project:
+    required: true
+  gcp_zone:
+    required: true
+  create:
+    required: false
+    description: Set to true to create a new cluster. The linkerd CLI should have been installed in the runner beforehand.
+  destroy:
+    required: false
+    description: Set to false to create a new cluster. The linkerd CLI should have been installed in the runner beforehand.
```

Comment:
Since this is not required, maybe this description should be `Set to true to destroy an existing cluster`?

Reply:
Yea, that description was off. I'm gonna end up merging that into an `action` field, as per Siggy's suggestion below.

Reply:
Yep sounds good I like that plan!